storage/engine/db.cc

```diff
@@ -27,9 +27,9 @@
 #include "rocksdb/options.h"
 #include "rocksdb/table.h"
 #include "rocksdb/utilities/write_batch_with_index.h"
-#include "cockroach/proto/api.pb.h"
-#include "cockroach/proto/data.pb.h"
-#include "cockroach/proto/internal.pb.h"
+#include "proto/api.pb.h"
+#include "proto/data.pb.h"
+#include "proto/internal.pb.h"
```

Comment:
What caused the includes to change? Not a big deal, but I prefer the `cockroach/` prefix.